package.json

```
@@ -86,16 +83,15 @@
     "eslint-plugin-unused-imports": "^3.0.0",
     "get-port": "^5.1.1",
     "immer": "^9.0.21",
-    "landlubber": "^1.0.0",
+    "landlubber": "^2.0.0",
     "mkdirp": "^3.0.0",
-    "next": "^12.3.4",
-    "nextjs-server-modules": "^4.6.1",
-    "nextlove": "^2.6.0",
     "prettier": "^3.0.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "tsup": "^8.0.1",
     "tsx": "^4.6.2",
-    "typed-axios-instance": "^3.1.0",
-    "typescript": "~5.1.0",
+    "typed-axios-instance": "^3.3.1",
+    "typescript": "~5.3.3",
```

Can go back to this now that upgrading is safe again
```suggestion
    "typescript": "^5.3.3",
```